Cubical/HITs/James/Inductive/PushoutFormula.agda

```diff
@@ -26,7 +26,7 @@ open import Cubical.Data.Sigma
 open import Cubical.HITs.Wedge
 open import Cubical.HITs.Pushout
 open import Cubical.HITs.Pushout.PushoutProduct
-open import Cubical.HITs.SequentialColimit
+open import Cubical.HITs.SequentialColimit renaming (incl to incl')
```

Comment:
I guess this is due to a name collision. I would prefer a name which clearly connects ```incl``` to ```SeqColim```, so it is clear on reading where it comes from.
Sometimes it is good to ```import``` something as, say ```SColim``` to later write ```SColim.incl```, but I don't know if it is a good idea here.

Reply:
Make sense. This is the only place where conflicting happens, I'll follow you here.